BaseListener/serverless.yml

```diff
@@ -0,0 +1,142 @@
+service: base-listener
+provider:
+  name: aws
+  runtime: dotnetcore3.1
+  memorySize: 2048
+  tracing:
+    lambda: true
+    apiGateway: true
+  vpc: ${self:custom.vpc.${opt:stage}}
+  stage: ${opt:stage}
+  account: ${opt:account}
+  region: eu-west-2
+
+package:
+  artifact: ./bin/release/netcoreapp3.1/base-listener.zip
```

Comment:
No.
I have modified the Renamer.ps1 file so that it is called with an additional parameter which replaces all instances of `base-listener`. See the readme in the root of the repo for instructions.